### PR TITLE
ISSUE-280: fixing path for project and repository apis

### PR DESCRIPTION
**Pull request by cristinabuenahora**
Issue: https://github.com/cdancy/bitbucket-rest/issues/280

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -27,6 +27,7 @@ dependencies {
     ext.guiceVersion = '5.0.1'
 
     implementation ("org.apache.jclouds:jclouds-core:${jcloudsVersion}")
+    implementation ("org.jfrog.buildinfo:build-info-extractor-gradle:4.21.0")
```
Comment (cdancy) [inline build.gradle:30]: @cristinabuenahora why this change?

Comment (cristinabuenahora) [reply build.gradle:30]: wouldn't compile otherwise - let me know if I was doing something wrong.

Comment (cdancy) [reply build.gradle:30]: @cristinabuenahora you should not need this line. The above change on line #4 should be enough to get things working.

Comment (cristinabuenahora) [reply build.gradle:30]: Ok, I'll remove and repush

**Comment by cdancy** [comment]
@cristinabuenahora will you need a new release or can you build/publish locally? Currently going through the process of getting this project hosted on maven central which may take a couple days.

**Comment by cristinabuenahora** [comment]
We can wait a few days - I'll need to test locally a bit more anyway. Thank you!!

**Comment by cristinabuenahora** [comment]
Hey Chris - I'm trying to write tests and mock the Bitbucket Client but I'm getting a "Cannot mock/spy class com.cdancy.bitbucket.rest.BitbucketClient" error - would you be open to making it non-final so that we can mock it?

**Comment by cristinabuenahora** [comment]
Never mind on my comment - I figured out how to write tests without needing to change it.

**Comment by cristinabuenahora** [comment]
Hey Chris - When do you think you'll be able to release this?

**Comment by cristinabuenahora** [comment]
Hey Chris - Hope you had a great long weekend. Just following up on this.

**Comment by cristinabuenahora** [comment]
Hey Chris - do you think you'll be able to merge this any time soon?

**Comment by cdancy** [comment]
My apologies as I've been off on other projects for the past 6 months and haven't had time to do any open-source work. Circling back on things now. I hope you're not still waiting on it. Taking it in regardless. Hopefully all is well your way. Thanks again!
